packages/sit-onyx/src/components/OnyxListbox/OnyxMiniSearch.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { computed, useAttrs, type HtmlHTMLAttributes } from "vue";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import xSmall from "@sit-onyx/icons/x-small.svg?raw";
+import { injectI18n } from "../../i18n";
+
+defineOptions({ inheritAttrs: false });
+
+const attrs = useAttrs();
+
+const props = defineProps<{ modelValue: string; label: string }>();
+
+const emit = defineEmits<{
+  "update:modelValue": [input: string];
+}>();
+
+const { t } = injectI18n();
+
+/**
+ * Current value (with getter and setter) that can be used as "v-model" for the native input.
+ */
+const value = computed({
+  get: () => props.modelValue,
+  set: (value) => emit("update:modelValue", value),
+});
+
+const rootAttrs = computed(
+  () =>
+    ({ class: attrs["class"], style: attrs["style"] }) as Pick<
+      HtmlHTMLAttributes,
+      "class" | "style"
+    >,
+);
+
+const inputAttrs = computed<Omit<HtmlHTMLAttributes, "class" | "style">>(() => {
+  const rest = { ...attrs };
+  delete rest.class;
+  delete rest.style;
+  return rest;
+});
+</script>
+<template>
+  <div class="onyx-mini-search" v-bind="rootAttrs">
+    <input
+      v-model="value"
+      :aria-label="props.label"
+      v-bind="inputAttrs"
+      class="onyx-mini-search__input"
+      placeholder="Search"
+      type="text"
```

Comment:
```suggestion
      type="search"
```

Type search also already has a clear icon that we can maybe style more easily and we might get right of the rotoAttrs and inputAttrs above

Reply:
The clear icon is User-Agent dependent and non-standard, see also: https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-search-cancel-button
Therefore I would like to stay with type `text`